libnmstate/ifaces/ifaces.py

```diff
@@ -165,6 +166,43 @@ def _validate_ovs_patch_peers(self):
                             " patch port"
                         )
 
+    def _validate_vlan_mtu_lte_base_mtu(self):
+        """
+            Validate that mtu of vlan or vxlan is less than
+            or equal to it's base interface's MTU
+
+            If base MTU is not present, set same as vlan MTU
+        """
+        for iface in self._ifaces.values():
+            if (
+                iface.iface_type == InterfaceType.VLAN
+                or iface.iface_type == InterfaceType.VXLAN
+            ):
+
+                if iface.state == InterfaceState.UP:
+
+                    if not iface.mtu:
+                        raise NmstateValueError(
+                            f"paramater mtu missing for vlan {iface.name}"
+                        )
+
+                    base_iface = self._ifaces.get(iface.parent)
+                    if not base_iface:
+                        raise NmstateValueError(
+                            f"base interface {iface.parent} for vlan \
+                                            {iface.name} does not exist"
+                        )
+
```

Comment:
This is my second time to say it:


This is not required.
The `Ifaces._validate_unknown_parent()` already checked it. No need to do it again.